### DISCUSSION TITLE
New Font

### DISCUSSION THREAD

**Discussion by Physignatily** [SummerFields/SummerFields#815] [Ideas]
Ok, this is something i wanted to talk about for quite some time now.

In the last years I suggested SummerFields to several people (friends, other random people on different servers). 
Most of the time they asked, after seeing some screenshots, then tried it for some time and really liked the textures, but stopped using it after some hours.

The main reason being, that the font is really hard on the eyes and makes ready chat or other text difficult.

And that's why I wanted to ask, if I could work on a new fontstyle for the pack.

**Comment by lipki** [comment]
With pleasure, honestly, we changed several times, it was never very clear this story of font.
You can take over the case.
